src/test/regress/expected/distributed_planning.out

```diff
@@ -537,6 +537,129 @@ ORDER BY 1;
 (4 rows)
 
 ROLLBACK;
+prepare p1 as INSERT INTO t1(a,c) VALUES (15, 15) ON CONFLICT (c) DO UPDATE SET a=EXCLUDED.a + 10 RETURNING *;
+execute p1(5);
+ a  | c
+---------------------------------------------------------------------
+ 15 | 15
+(1 row)
+
+execute p1(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 15
+(1 row)
+
+execute p1(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 15
+(1 row)
+
+execute p1(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 15
+(1 row)
+
+execute p1(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 15
+(1 row)
+
+execute p1(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 15
+(1 row)
+
+prepare p5(int) as INSERT INTO t1(a,c) VALUES (15, $1) ON CONFLICT (c) DO UPDATE SET a=EXCLUDED.a + 10 RETURNING *;
+execute p5(5);
+ a  | c
+---------------------------------------------------------------------
+ 15 | 5
+(1 row)
+
+execute p5(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 5
+(1 row)
+
+execute p5(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 5
+(1 row)
+
+execute p5(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 5
+(1 row)
+
+execute p5(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 5
+(1 row)
+
+execute p5(5);
+ a  | c
+---------------------------------------------------------------------
+ 25 | 5
+(1 row)
+
+INSERT INTO "companies" ("id","meta","name","created_at","updated_at","deleted_at") VALUES (1,'{"test":123}','Name','2016-11-07 17:34:22.101807','2021-05-20 22:16:55.424521',NULL) ON CONFLICT (id) DO UPDATE SET "meta"=EXCLUDED."meta"  RETURNING "id";
+ id
+---------------------------------------------------------------------
+  1
+(1 row)
+
+INSERT INTO "companies" ("id","meta","name","created_at","updated_at","deleted_at") VALUES (1,'{"test":123}','Name','2016-11-07 17:34:22.101807','2021-05-20 22:16:55.424521',NULL) ON CONFLICT (id) DO UPDATE SET "meta"=EXCLUDED."meta"  RETURNING "id";
+ id
+---------------------------------------------------------------------
+  1
+(1 row)
+
+PREPARE p6 AS INSERT INTO "companies" ("id","meta","name","created_at","updated_at","deleted_at") VALUES (1,'{"test":123}','Name','2016-11-07 17:34:22.101807','2021-05-20 22:16:55.424521',NULL) ON CONFLICT (id) DO UPDATE SET "meta"=EXCLUDED."meta"  RETURNING "id";;
```

Comment:
Although we currently do not support INSERT .. SELECT through the same code-path here: Can you add a similar test with `INSERT .. SELECT .. ON CONFLICT .. RETURNING`?

```SQL
prepare insert_select(int) as insert into companies SELECT * FROM companies WHERE id >= $1 ON CONFLICT(id) DO UPDATE SET "meta"=EXCLUDED."meta" RETURNING "id";;
```